packages/internal/src/__tests__/viteNestedPages.test.mts

```diff
@@ -0,0 +1,241 @@
+import assert from 'node:assert/strict'
+import path from 'path'
+import { fileURLToPath } from 'url'
+
+import { after, before, beforeEach, describe, it } from 'node:test'
+
+import projectConfig from '@redwoodjs/project-config'
+import { vitePrebuildWebFile } from '@redwoodjs/vite'
```

Comment:
This isn't great.
`@redwoodjs/vite` already depends on `@redwoodjs/internal`, so this makes the dependency graph circular. It "works" because `@redwoodjs/vite` isn't listed as a dependency of internal, but clearly it is now.

Reply:
Moved the test to the vite package